### ISSUE TITLE
Update the "Community" link(s) in the ToC panel

### ISSUE THREAD

**Issue by TibsAtWork** [aiven/devportal#1749]
We now have a Community page at aiven.io (http://avien.io/community) and should reflect this in the documentation.

One possibility is to replace the current "Community" link with a direct link to that page,
and then move the existing ToC entries into a "How to Contribute" clause. The two challenges could go under a new "Challenges" item.

We should consider trying out some layouts to see what works.

**Comment by angelinekwan** [comment]
@TibsAtWork Is this something we can try in a PR already or ...

**Comment by TibsAtWork** [comment]
@angelinekwan Yes, definitely - a new branch to try what it looks like and a corresponding PR is the right way to go.
It probably won't be me in the near term, but someone else may pick it up.

**Comment by harshini-rangaswamy** [comment]
FYI @TibsAtWork - This [PR](https://github.com/aiven/devportal/pull/1946) is in progress to include the Community information in the docs.
